test/e2e/apps/deployment.go

```diff
@@ -880,12 +884,19 @@ func testProportionalScalingDeployment(f *framework.Framework) {
 	err = framework.WaitForReplicaSetDesiredReplicas(c.ExtensionsV1beta1(), secondRS)
 	Expect(err).NotTo(HaveOccurred())
 
-	framework.Logf("Verifying that first rollout's replicaset has 11 available replicas")
-	err = framework.WaitForReplicaSetAvailableReplicas(c.ExtensionsV1beta1(), firstRS, 11)
+	numReplicaDiff := newReplicas - replicas
+	if numReplicaDiff < 0 {
```

Comment:
To clarify the formula, `newReplicas - replicas` is difference between old and new number of replicas after a scaling event. To cater for both scaling _up_ and _down_ events, we need to check the case when `numReplicaDiff < 0`.